plume-models/src/blogs.rs

```diff
@@ -133,10 +133,8 @@ impl Blog {
     pub fn find_by_fqn(c: &PlumeRocket, fqn: &str) -> Result<Blog> {
         let from_db = blogs::table
             .filter(blogs::fqn.eq(fqn))
-            .limit(1)
-            .load::<Blog>(&*c.conn)?
-            .into_iter()
-            .next();
+            .first(&*c.conn)
+            .optional()?;
```

Comment:
First return a `Result<Blog, _>`, [optional](https://docs.diesel.rs/diesel/result/trait.OptionalExtension.html) convert it to a `Result<Option<Blog>,...>` so that if the error is not found, we can fetch the blog instead of not knowing it ever, but if the error is anything else, it's still an error

Reply:
thank you for the explanation!